calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -22,17 +28,21 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Before you begin...
 
-The following platforms:
+**Unsupported**
+
+- GKE
```

Comment:
IMHO, switching from including all supported platforms to excluding unsupported ones is complicated, I don't claim to know this for certain and would ask @mgleung, @caseydavenport and/or @lwr20, would you please chime in regarding platforms we support (Calico in general, in addition to wireguard)?

My point being we're going from claiming to support Kubernetes onprem, EKS and AKS, to only not supporting GKE (and that might include things we're not anticipating). Again, I'm not the most knowledgeable in this, but aren't there other non-kubernetes platforms, for example? (on a tangent, besides k8s onprem, don't we support k8s + clouds such as gcp, aws and azure? or are those considered as the same as onprem?)

Sorry about the braindump here, I feel like I should be able to answer at least some of these questions (and I'll investigate documentation+code to do so), but I wanted to start a discussion...

Reply:
I agree this is a concern.  The thing about OSS is that new platforms pop up all the time, and users need to understand what to expect with them.  How can we possibly list the infinity of things we do not support when they're being created all the time?  

Its better to state the platforms we test regularly, so that users can know which platforms will "just work".  
Also useful to state platforms where we know it cannot work so they don't waste time trying.

Reply:
@bmckercher123 
Including Barbara in the discussion.

Reply:
You raise a good point about OSS and what "support" means. The doc was written from the perspective of Enterprise, which has changed to documenting the handful of features that we know do not work, or have significant concerns. For Enterprise and OSS, we cannot use the measure of "what we test regularly" because we simply do not test everything every release. Right now, we are following what we've done since the beginning by aligning the doc sets. I've created a ticket to revisit this with PM: https://tigera.atlassian.net/jira/software/projects/DOCS/boards/69/backlog?selectedIssue=DOCS-1072.

Reply:
Agree with @coutinhop and @lwr20.

Reply:
> For Enterprise and OSS, we cannot use the measure of "what we test regularly" because we simply do not test everything every release. 

No, we don't test every possible combination, but I think that's _expected_ in OSS.  What I'm proposing here is that we state what we test regularly and what we _know_ doesn't work, so that users can know how easy/difficult/impossible it will be to use whatever platform they're thinking of using.